onboarding-checklist.md

```diff
@@ -0,0 +1,140 @@
+# Onboarding Checklist #
+
+## Instructions ##
+
+When someone new joins the team:
+
+1. A team member adds the new team member to the `cisagov` organization in
+GitHub.
+1. Create a new issue called "Onboard _Name_".
+1. View the raw source of this file.
+1. Copy the text after the line below to the issue.
+1. Remove any block of tasks that doesn't apply to the new team member's role
+1. Submit the issue.
+1. Assign the new person and their onboarding buddy to the issue.
+1. Move the issue into _In Progress_.
+
+---
+
+## Directions ##
+
+**Onboardee and buddy:** Try to go through your checklists in order.
+
+**Buddy:** If you can’t complete any of the items on your checklist
+personally, you are responsible for ensuring that someone with the correct
+access completes that item.
+
+## Onboardee checklist ##
+
+### Getting to know VM Fusion ###
+
+- [ ] Take notes on anything that is confusing or frustrating during your
+onboarding process.
+  - Please share this information with your buddy and/or the team so we can
+  make things better.
+  - If you notice a problem (especially with things like documentation), you
+  are more than welcome to fix it!
+  - You can also file issues and pull requests on the [template onboarding checklist]
+- [ ] Meet with your onboarding buddy (they should reach out to you).
+- [ ] Once you've finished the checklists below, make suggestions for steps
+that would have improved your onboarding experience as pull requests on the
+[template onboarding checklist] used to make this issue.
+
+### Required items for all team members ###
+
+Completing these items helps us fulfill security and compliance requirements.
+If you get stuck, or if these requirements are confusing, ask for help from
+your onboarding buddy.
+
+Pre-requisites:
+
+- [ ] Complete [PALMS Cybersecurity Training](https://etms.hq.dhs.gov),
+including Mandatory Cyber Security and Privacy Training, including accepting
+the Rules of Behavior, which is required before we can give you access to any
+systems, like Trio.
+- [ ] Go through the [FISMA-Ready Github guide](https://github.com/fisma-ready/github)
+to set up your Github account to be added to CISA
+
+### Microsoft Teams chat ###
+
+Once you're set up on Microsoft Teams, we recommend you join the following:
+
+- [ ] [CISA](https://teams.microsoft.com/l/team/19%3aa40fd338ea7a45de994a7c7ce9671fd4%40thread.skype/conversations?groupId=8c6c0b00-7244-46e1-bef9-76ce2a1906f5&tenantId=3ccde76c-946d-4a12-bb7a-fc9d0842354a)
+- [ ] [CISA - VM- Fusion Branch](https://teams.microsoft.com/l/team/19%3a2d4e9f3e321f4ef9a09f4ed94a32105c%40thread.skype/conversations?groupId=2992771f-df53-4097-87dd-75898a5830db&tenantId=3ccde76c-946d-4a12-bb7a-fc9d0842354a)
+
+### Getting to know the VM Fusion Dev Team ###
+
+- [ ] Bookmark the [dev team project board] to track issues and work across
+multiple Github repositories
+- [ ] Bookmark [VM JIRA] to work with the rest of VM Fusion.
```

Comment:
This bullet is a complete sentence but does not end in a period, whereas other bullets in this block that are complete sentences do.  We should be consistent, at least in the same bulleted list.

I'm for adding a period if the bullet is a complete sentence, but as long as we are globally consistent I'm OK with what the group decides.

Reply:
Resolved a848650149665216a068543dd1b50962d7ec2737